project.clj

```diff
@@ -7,11 +7,12 @@
                    :dependencies   [[org.slf4j/slf4j-api        "1.7.25"]
                                     [org.slf4j/slf4j-log4j12    "1.7.25"]
                                     [org.clojure/tools.logging  "0.4.0"]]}}
+  :plugins [[lein-cljfmt "0.5.7"]]
```

Comment:
I'd keep this into your `.lein/profiles.clj`

Here's mine.

```clj
{:user
 {:dependencies  [[cljfmt "0.5.1"]]
  :plugins       [[ lein-ancient "0.6.15"]]
  :repl-options  {:init (require 'cljfmt.core)}}
 :auth
 {:repository-auth
  {#"artifacts.exoscale.ch" {:username "·····"
                             :password "·····························"}}}}

```

Reply:
Ah nice, good point.

:+1: